platform/golden/cfg/preseed-debian-11.cfg

```diff
@@ -1,95 +1,142 @@
+d-i passwd/user-fullname string ArtificialWisdom
+d-i passwd/username string packer
+d-i passwd/user-password password insecure
+d-i passwd/user-password-again password insecure
+d-i passwd/user-default-groups string users sudo
 
 
-###################
-## Select packages
+####
+## B.4.6 Clock and time-zone setup
 
-tasksel tasksel/skip-tasks  string standard
-tasksel tasksel/first multiselect openssh-server
-popularity-contest popularity-contest/participate boolean false
+d-i clock-setup/utc boolean true
+d-i time/zone string UTC
+d-i clock-setup/ntp boolean true
 
-d-i pkgsel/include string curl wget openssh-server neovim sudo cloud-init
-d-i pkgsel/language-pack-patterns   string
-d-i pkgsel/install-language-support boolean false
-d-i pkgsel/upgrade select full-upgrade
 
-###########
-## Use XFS
+####
+## B.4.7 Partitioning
+## Our cloud requires the use of xfs
 
 d-i partman/default_filesystem string xfs
 d-i partman-auto/method string regular
 
-###################
-## Volume defaults
+
+####
+## B.4.7.aw0 Volume defaults
 
 d-i partman-auto/choose_recipe select atomic
 d-i partman-partitioning/confirm_write_new_label boolean true
+d-i partman-basicfilesystems/no_swap boolean true
 d-i partman/choose_partition select finish
 d-i partman/confirm boolean true
 d-i partman/confirm_nooverwrite boolean true
 
-####################
-## Additional media
+
+####
+## B.4.7.aw1 Boot defaults
+
+d-i partman-efi/non_efi_system boolean true
+d-i partman-partitioning/choose_label select gpt
+d-i partman-partitioning/default_label string gpt
+
+
+####
+## B.4.7.3 Controlling how partitions are mounted
+
+d-i partman/mount_style select uuid
+
+
+####
+## B.4.8 Base system installation
+
+d-i base-installer/install-recommends boolean true
+d-i base-installer/initramfs-tools/driver-policy select generic
+d-i base-installer/kernel/image select linux-image-amd64
+
+
+####
+## B.4.9 Apt setup
+
 d-i apt-setup/cdrom/set-first boolean false
-d-i apt-setup/cdrom/set-next boolean false
-d-i apt-setup/cdrom/set-failed boolean false
-d-i apt-setup/non-free boolean false
-d-i apt-setup/contrib boolean false
+d-i apt-setup/disable-cdrom-entries boolean true
 
-#######################
-## Bootloader defaults
 
-d-i grub-installer/only_debian boolean true
-d-i grub-installer/bootdev  string default
+####
+## B.4.10 Package selection
+
+tasksel tasksel/first multiselect standard, ssh-server
+d-i pkgsel/include string sudo cloud-init curl open-iscsi libopeniscsiusr openssh-server neovim
+d-i pkgsel/upgrade select full-upgrade
+popularity-contest popularity-contest/participate boolean false
 
-#####################
-## Add late commands
 
-d-i preseed/late_command string echo "packer ALL=(ALL:ALL) NOPASSWD:ALL" > /target/etc/sudoers.d/packer && chmod 0440 /target/etc/sudoers.d/packer
+####
+## B.4.11 Boot loader installation
 
-#######################
-## finish
+d-i debian-installer/add-kernel-opts console=tty1 console=ttyS0,115200
```

Comment:
I am considering changing this in a follow-up PR such that kernel options are added in /etc/default/grub.d/option.cfg, rather than /etc/default/grub as this `di` change does.

Reply:
Note that the grub docs (https://www.gnu.org/software/grub/manual/grub/grub.html#Simple-configuration) point to simple changes like adding a console configuration, are valid in /etc/default/grub, so this doesn't _have_ to be changed unless there is some other requirement that drives this.